ImageFeed/SplashViewController.swift

```diff
@@ -6,12 +6,17 @@
 //
 
 import UIKit
+import ProgressHUD
 
 final class SplashViewController: UIViewController {
 
     private var isUserAuthorized = false
     private let mainStoryboard = UIStoryboard(name: "Main", bundle: nil)
     private lazy var tokenStorage: OAuth2TokenStorageProtocol = OAuth2TokenStorage()
+    private lazy var oAuth2Service: OAuth2ServiceProtocol = OAuth2Service()
```

Comment:
Обычно реализация сервиса имеет более конкретное название, а протоколы (интерфейсы) более общие. Так ты поддерживаешь концепцию, что тебе важен результат работы сервиса, а не то как именно он это делает. Предлагаю протокол Описать короче AuthServiceProtocol и переменную называть соответствующе
```suggestion
    private lazy var authService: OAuth2ServiceProtocol = OAuth2Service()
```